pom.xml

```diff
@@ -364,64 +364,24 @@
             </snapshots>
         </repository>
 
-        <!--
-           P2 Repositories for tycho build.
-           This list should match or at least be a subset of the list of URL's in openhab-distro target platform.
-           These repos can be mirrored by maven if needed.
-        -->
+        <!-- SmartHome p2 repository -->
         <repository>
             <id>p2-smarthome</id>
             <url>http://download.eclipse.org/smarthome/updates-stable</url>
             <layout>p2</layout>
         </repository>
+
+        <!-- openHAB core p2 repository -->
         <repository>
             <id>p2-openhab-core</id>
             <url>https://dl.bintray.com/kaikreuzer/openhab-core/p2/${ohc.version}</url>
             <layout>p2</layout>
         </repository>
+
+        <!-- openHAB platform p2 repository -->
         <repository>
-            <id>p2-eclipse-orbit</id>
-            <url>http://download.eclipse.org/tools/orbit/downloads/drops/R20140525021250/repository</url>
-            <layout>p2</layout>
-        </repository>
-        <repository>
-            <id>p2-eclipse-modeling</id>
-            <url>http://download.eclipse.org/modeling/tmf/xtext/updates/releases/maintenance/R201407030533</url>
-            <layout>p2</layout>
-        </repository>
-        <repository>
-            <id>p2-eclipse-luna</id>
-            <url>http://download.eclipse.org/releases/luna/201502271000</url>
-            <layout>p2</layout>
-        </repository>
-        <repository>
-            <id>p2-eclipse-jetty</id>
-            <url>http://download.eclipse.org/jetty/updates/jetty-bundles-9.x/9.2.12.v20150709</url>
-            <layout>p2</layout>
-        </repository>
-        <repository>
-            <id>p2-jax-rs</id>
-            <url>http://hstaudacher.github.io/osgi-jax-rs-connector/5.2.0</url>
-            <layout>p2</layout>
-        </repository>
-        <repository>
-            <id>p2-groovy</id>
-            <url>http://dist.springsource.org/release/GRECLIPSE/e4.4</url>
-            <layout>p2</layout>
-        </repository>
-        <repository>
-            <id>p2-jupnp</id>
-            <url>http://www.jupnp.org/update-site</url>
-            <layout>p2</layout>
-        </repository>
-        <repository>
-            <id>p2-paho</id>
-            <url>http://download.eclipse.org/paho</url>
-            <layout>p2</layout>
-        </repository>
-        <repository>
-            <id>p2-jmdns</id>
-            <url>http://www.jmdns.org/update-site/3.4.2</url>
+            <id>p2-openhab-platform</id>
+            <url>http://repository-openhab.forge.cloudbees.com/release/openhab-platform-p2/${tp.version}</url>
```

Comment:
Having the p2 platform repo on cloudbees does seem to make a big difference in duration. My latest [pull request test] (https://openhab.ci.cloudbees.com/view/Pull%20Request%20Builds/job/PR-openHAB1-Addons/23/) for openhab1 addons finished in 13 minutes...